2_adventures/001/Randhir123/05/lib/05/fibonacci.rb

```diff
@@ -0,0 +1,29 @@
+class Fibonacci
+	
+	def initialize()
+		@cache = {}	
+	end
+	
+	def get(n)
+		if n == 0 || n == 1
+			return n
+		else
+			return get(n-2) + get(n-1)
+		end
```

Comment:
return is not needed in either place.  The reason is that Ruby returns the last line evaluated in a method, and the last line *is* the if statement.

Also, you don't do the same thing for get_optimized, so I can say "You already know this!".